client/src/components/authentication/Auth.js

```diff
@@ -1,96 +1,88 @@
-import history from '../../history';
-import auth0 from 'auth0-js';
+import Auth0Lock from 'auth0-lock';
+import axios from 'axios';
+
+const clientID = 'jNDq5B6iAnIRcrpM07Omh05uyppZ89px';
+const domain = 'team-refreshr.auth0.com';
+const options = {
+  auth: {
 
-// ...
-export default class Auth {
-  accessToken;
-  idToken;
-  expiresAt;
-  auth0 = new auth0.WebAuth({
-    domain: 'team-refreshr.auth0.com',
-    clientID: 'jNDq5B6iAnIRcrpM07Omh05uyppZ89px',
-    audience: 'https://team-refreshr.auth0.com/api/v2/',
     //PRODUCTION
+    //audience: 'https://refreshr.herokuapp.com',
+    //redirect: true,
     // redirectUri: 'https://refreshr-app.netlify.com/loading',
     //DEVELOPMENT
-    redirectUri: 'http://localhost:3000/loading',
-
-    responseType: 'token id_token',
-    scope: 'openid'
-  });
-
-  login = () => {
-    this.auth0.authorize();
-  };
-
-  handleAuthentication = () => {
-    this.auth0.parseHash((err, authResult) => {
-      if (authResult && authResult.accessToken && authResult.idToken) {
-        this.setSession(authResult);
-        localStorage.setItem('token', authResult);
-
-        // history.replace('/home')
-      } else if (err) {
-        history.replace('/');
-        console.log(err);
-        alert(`Error: ${err.error}. Check the console for further details.`);
-      }
-    });
-  };
+    audience: 'http://localhost:9000',
+    redirect: true,
+    redirectUrl: 'http://localhost:3000/dashboard',
+    usernameStyle: 'email',
+    responseType: 'token id_token'
+  },
+  socialButtonStyle: 'small',
+  theme: {
+    primaryColor: '#0b2742',
+    backgroundColor: '#000',
+    logo: 'http://i67.tinypic.com/vfgxf8.png'
+  },
+  loginAfterSignup: true,
+  additionalSignUpFields: [
+    {
+      name: 'first_name',
+      placeholder: 'First Name',
+      icon: 'http://i63.tinypic.com/2z886rs.png'
+    },
+    {
+      name: 'last_name',
+      placeholder: 'Last Name',
+      icon: 'http://i63.tinypic.com/2z886rs.png'
+    }
+  ]
+};
+const lock = new Auth0Lock(clientID, domain, options);
 
-  getAccessToken = () => {
-    return this.accessToken;
-  };
+lock.on('authenticated', authResult => {
+  // Use the token in authResult to getUserInfo() and save it to localStorage
+  lock.getUserInfo(authResult.accessToken, function(error, profile) {
+    if (error) {
+      alert(error);
+      return;
+    }
+    //console.log(authResult);
 
-  getIdToken = () => {
-    return this.idToken;
-  };
+    localStorage.setItem('accessToken', authResult.accessToken);
+    localStorage.setItem('profile', JSON.stringify(profile));
+    localStorage.setItem('name', `${profile.name}`)
+    localStorage.setItem('user_id', profile['http://localhost:9000/uid']);
 
```

Comment:
This is awesome, seems like a much more straightforward way to do auth than what we had before.